.github/workflows/ui-test.yml

```diff
@@ -79,13 +79,19 @@ jobs:
         run:  |
           echo 'UI_URL=https://staging.mangata.finance/' >> $GITHUB_ENV
           echo 'API_URL=wss://staging.mangatafinance.cloud:9944/' >> $GITHUB_ENV
-  
+          echo 'ETHEREUM_WEBSOCKET_URL=ws://localhost:8545' >> $GITHUB_ENV
```

Comment:
Hey @eddyjlsh ,
I can see 3 issues,
* Integration config is missing.
* Values are duplicated for develop and staging.
* Values does not match with the zip that contains all this info. Am I missing anything?